### PR TITLE
✨  Update readme

### PR DESCRIPTION
**Pull request by laurentsimon**
* **Please check if the PR fulfills these requirements**
- [] Tests for the changes have been added (for bug fixes / features)
- [] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
doc update

* **What is the current behavior?** (You can also link to an open issue here)
missing checks

* **What is the new behavior (if this is a feature change)?**
adding missing checks

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,9 @@ Active             | Did the project get any commits in the last 90 days?
 Branch-Protection  | Does the project use [Branch Protection](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/about-protected-branches) ?
 Packaging          | Does the project build and publish official packages from CI/CD, e.g. [GitHub Publishing](https://docs.github.com/en/free-pro-team@latest/actions/guides/about-packaging-with-github-actions#workflows-for-publishing-packages) ?
 Vulnerabilities    | Does the project have unfixed vulnerabilities? Uses the [OSV service](https://osv.dev).
+Token-Permissions  | Does the project declare GitHub workflow tokens as [read only](https://docs.github.com/en/actions/reference/authentication-in-a-workflow)?
+Automatic-Dependency-Update | Does the project use tools to automatically update its dependencies?
+Binary-Artifacts   | Is the project free of checked-in binaries?
```
Comment (inferno-chromium) [inline README.md:65]: Thanks a ton. Can you sort the list of checks and then just commit the PR.

**Review by inferno-chromium — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (laurentsimon) [reply README.md:65]: sort alphabetically?

Comment (naveensrinivasan) [reply README.md:65]: Yes, sort alphabetically! I was thinking of doing that. Thanks

Comment (inferno-chromium) [reply README.md:65]: yes :)

**Pull request merged**